scripts/build/build-docs.js

```diff
@@ -24,7 +24,7 @@ const parserPaths = parsers.map(stripLanguageDirectory);
 
 if (isPullRequest) {
   const pkg = require("../../package.json");
-  pkg.prVersion = `pr-${process.env.REVIEW_ID}`;
+  pkg.version = `999.999.999-pr.${process.env.REVIEW_ID}`;
```

Comment:
Why not use `[current version]-pr.1234`?

Reply:
I think a version followed by `-something` is meant to denote pre-release versions (which is not the case)

Reply:
Ref #3433 and #3573

https://github.com/prettier/prettier/blob/7eca8011bc5e6f6071e94116619b3fda377befbb/src/common/support.js#L12-L17

We use this version to determine which `language` (and `option` in the future PR) can be used, which means it should be greater than the current version in PR builds so we can test it on the preview playground.